### PR TITLE
Show a tooltip on hovering over the instrument viewer

### LINKED ISSUE #33567: Show a tool tip on hovering in the instrument viewer
At the request of some scientists at ILL, when hovering over the instrument in the instrument viewer, a tool tip showing basic information about the pixel should appear in every mode and tab, instead of forcing the user to go back to the pick tab and select the correct mode every time. It is mostly in demand for rectangular detectors.

This tool tip would only show the detector name - which in rectangular detectors also contains its position - , the name of its parents - the panel it's part of - and the integrated counts.

There should be an option int he settings to activate this feature.

### PR DESCRIPTION
**Pull request by MathieuTillet**
**Description of work.**
At the request of some scientists at ILL, when hovering over the instrument in the instrument viewer, a tool tip showing basic information about the pixel appears. It is mostly useful and meant for rectangular detectors.

This tool tip only shows the detector name - which in rectangular detectors contains all relevant info about its position - and the integrated counts.

This feature is of course optional and can be activated via the settings menu.

**To test:**
- Load a file with a rectangular detector - for example `ExternalData/Testing/Data/UnitTest/ILL/D22B/046600.nxs`
- Open the instrument viewer.
- In the `Display settings` menu, check `Tooltip`
- Hover over the instrument and check the tool tip is there.

Closes #33567 .

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.4.0/mantidworkbench.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 - New general setting: `Provide auto-completion suggestions in the script editor` for making auto-completion optional in the script editor
+- Nex option in the instrument viewer to see basic detector data in a tooltip when hovering.
```
Comment (ConorMFinn) [inline docs/source/release/v6.4.0/mantidworkbench.rst:11]:
```suggestion
- New option in the instrument viewer to see basic detector data in a tooltip when hovering.
```

**Review by cailafinn — CHANGES_REQUESTED** [review]
Works well, code-wise just a tiny note on the release note. 

Also noticed that if the view type is switched, e.g. from full 3D to any of the others, the checkbox in Display Settings remains ticked but no tooltip appears.

**Comment by MathieuTillet** [comment]
Thanks for catching that, I should work well now!

**Review by cailafinn — APPROVED** [review]
Looks good to me now, nice work 👍

**Review by DavidFair — APPROVED** [review]
LGTM and makes a big difference to the UX not having to move your eyes back and forth to see the same information

**Pull request merged**